README.md

```diff
@@ -255,23 +255,51 @@ Registry](https://github.com/stackbuilders/hapistrano/pkgs/container/hapistrano)
 
 Check the documentation [here](.github/workflows/README.md)
 
-## Nix
+## Development
```

Comment:
@sestrella Please, don't forget to update the table of contents if you update the README titles. Thanks!

Reply:
@CristhianMotoche what do you think about getting rid of it, since GH auto-generates one?

<img width="668" alt="image" src="https://user-images.githubusercontent.com/2049686/221959850-95ce74de-2d0d-47e7-9977-a4429858e6bf.png">

Reply:
Sounds good to me @sestrella